README.md

```diff
@@ -28,24 +28,22 @@ If you want to change the look and feel of the ownCloud documentation, you can f
 
 ## How To Generate PDFs
 
-To generate a PDF version of one or more of the manuals, you need to have `asciidoctor-pdf` installed.
-With it installed, run the command below, in the root docs directory, to generate a PDF version of a manual.
-The PDF file will be generated in the same directory as where the command is run, and will be named after the configuration file, with a `.pdf` extension.
+To generate the PDF copies of the three manuals (administration, developer, and user), you need to have `asciidoctor-pdf`, and GNU make installed installed.
+If they’re installed, run the command below in the root directory, to generate the manuals.
 
 ```console
-asciidoctor-pdf \
-    book.adoc \
-    -a pdf-stylesdir=resources/themes \
-    -a pdf-style=owncloud
+make pdf
 ```
 
+The PDF files will be generated in the `build` directory, and will be named after the configuration file.
```

Comment:
``generated in the `build` directory`` is located where?

Reply:
The intent of formatting "build" as inline code, was to refer to the directory of that same name. I see that it's not as clear as I'd intended and make the directory name that much more explicit.